_posts/2020-09-28-argentina-big-data-meetup.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title:  "Presto at Argentina Big Data Meetup 2020-09-23"
+author: Brian Olsen
+excerpt_separator: <!--more-->
+---
+
+{% youtube IkjNcW7cS2w %}
+
+Martin made a guest appearance at the 
+[Argentina Big Data Meetup](https://www.meetup.com/Argentina-Big-Data-Meetup/)
+(online) where in the first hour Martin talks about Presto's past, present, and
+future. This includes the history from Facebook to Starburst, some context to
+some early architectural decisions, as well as, why Presto was open-sourced. 
+Finally, Martin covers recent changes along with some upcoming changes on the
+roadmap.
+
+[Slides](/assets/blog/argentina-big-data-meetup/Presto%20-%20Big%20Data%20Meetup%20Argentina%202020-09-23.pdf)
+
+The next hour is an interesting talk given by Federico Palladero covering his
+company, Jampp's, migration strategy from EMR Presto to Docker using Nomad vs
+Kubernetes.
+
+[Slides](/assets/blog/argentina-big-data-meetup/Big%20Data%20Meetup_%20Presto%20on%20Docker.pdf)
+
+These presentations are in Spanish
```

Comment:
Missing period at the end of the sentence.

Reply:
done